frontend/src/global_styles/common/header/logo.sass

```diff
@@ -1,12 +1,11 @@
 .op-logo
-  width: 230px
-  height: var(--header-height)
+  display: flex
+  margin: 10px 0
+  min-width: 230px
 
   &--link
-    margin-top: 12px
     width: 100%
     display: block
-    height: 30px
```

Comment:
I think we should keep the height restriction as it enforces the logo to be horizontally aligned with the other elements of the tooolbar which looks cleaner imho.

Reply:
Hi @HDinger, yes agree, when we set the height for the logo, it is aligned with other header elements. 👍 
I thought if we don't set a height for it, it will stretch to the whole height of header and will be bigger.